acquisitions/src/main/resources/thunderjet/consortia/tenant-utils/consortia-admin-verification-and-setup.feature

```diff
@@ -0,0 +1,180 @@
+Feature: verify records for real/shadow 'consortiaAdmin' and make him consortia wide admin
```

Comment:
we can delete this file as well, orders tests should be executed without it

Reply:
removed